process/subprocesses/ghsci.py

```diff
@@ -797,15 +793,13 @@ def _region_data_setup(
                     f"{region}.yml error: The 'data_dir' entry for {data} does not appear to have been defined.  This parameter is required for analysis of {region}, and is used to locate a required dataset cross-referenced in {region}.yml.  Please check the configured settings before proceeding.",
                 )
                 return None
-            if data_path is not None and not data_dictionary[
-                'data_dir'
-            ].startswith('Not required'):
+            if data_path is not None:

```

Comment:
As above, I am wary of changes to this logic if there is not a reason for it.  The checks here were developed after testing a range of edge cases.